gradle.properties

```diff
@@ -109,7 +109,7 @@ jetbrains.compose.compiler.version=1.5.14.1
 #   (artifactRedirection.version.androidx.compose covers androidx.compose.*:*)
 artifactRedirection.version.androidx.compose=1.9.0-rc01
 artifactRedirection.version.androidx.compose.material.material-navigation=1.9.0-rc01
-artifactRedirection.version.androidx.compose.material3=1.4.0-alpha18
+artifactRedirection.version.androidx.compose.material3=1.4.0-beta01
 artifactRedirection.version.androidx.compose.material3.adaptive=1.2.0-alpha09
 artifactRedirection.version.androidx.compose.material3.common=1.0.0-alpha01
```

Comment:
It looks like we forgot to remove it in the previous PR. We might do that in this or in a separate PR

Reply:
Or just align it during adoption new `buildSrc`